CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @planningcenter/people
```

Comment:
_Technically_ People doesn't even use this gem anymore. We've switched to the official Mailchimp client. The code is still in use in Accounts though. I don't particularly mind People being a codeowner, especially since we are the consumers of the feature, but It might make sense to add `@planningcenter/giving` also, since any changes here would impact that product.

Reply:
Added but can you add the correct permissions?